packages/devtools_app/lib/src/screens/memory/panes/leaks/diagnostics/formatter.dart

```diff
@@ -16,9 +16,9 @@ String analyzedLeaksToYaml({
 }) {
   return '# For memory leaks troubleshooting tips see\n'
       '# $linkToGuidance\n\n'
-      '${LeakReport.iterableToYaml('not-disposed', notDisposed)}'
+      '${LeakReport.iterableToYaml('not-disposed', notDisposed, phasesAreTests: false)}'
```

Comment:
@polina-c can you please review the changes to this file? is `false` the proper value for all these method calls?

Reply:
yes, thank you